Documentation/ApiOverview/Fluid/_CustomViewHelper/_GravatarTagBasedViewHelper_getContentArgumentName.php

```diff
@@ -26,4 +31,9 @@ public function render(): string
 
         return $this->tag->render();
     }
+
+    public function getContentArgumentName(): string
```

Comment:
This method is not used

Reply:
This is another new feature btw: https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/13.3/Feature-104789-SupportForContentArgumentNameInAbstractViewHelper.html#feature-104789-support-for-contentargumentname-in-abstractviewhelper

Reply:
OK. THX